### PR TITLE
Issue 6353 implement Windows ordering buttons ("OK" / "Cancel")

### PR DESCRIPTION
**Pull request by zyadtaha**
Fixes [#6353](https://github.com/OpenRefine/OpenRefine/issues/6353)

Changes proposed in this pull request:
- Implemented Windows ordering buttons ("OK" / "Cancel")
- Marked the OK button  as "primary" 

Here's how it looks like :
Screenshot from 2024-03-24 13-29-28

### CODE REVIEW

**Review by wetneb — COMMENTED** [review]
Very nice, thank you! I wonder if you considered re-using the existing CSS class `button-primary`? If so, why did you choose to introduce `okButton` instead?

**Comment by zyadtaha** [comment]
Yes, you are right it's better to reuse the existing class.
Sorry for the mistake, had fixed it.

**Review by wetneb — COMMENTED** [review]
I have had a quick look and the migration looks complete (although there can be places I have overlooked too).
I like that the order we went for seems to be consistent with the majority of existing dialogs so it shouldn't come at a big surprise to users.

File: extensions/wikibase/module/scripts/dialogs/wikibase-dialog.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
     </div>
     <div class="dialog-footer-bg">
       <button class="button" bind="cancelButton"></button>
-      <button class="button" bind="okButton"></button>
+      <button class="button button-primary" bind="okButton"></button>
```
[the remark this answers is not in the document]
Comment (zyadtaha) [reply extensions/wikibase/module/scripts/dialogs/wikibase-dialog.html:17]: Done.

**Review by wetneb — APPROVED** [review]

File: main/webapp/modules/core/styles/reconciliation/recon-dialog.css [hunk 2]
```diff
@@ -104,7 +104,6 @@ a.recon-dialog-service-selector-remove {
 }
 .dialog-footer {
```
Comment (tfmorris) [inline main/webapp/modules/core/styles/reconciliation/recon-dialog.css:105]: Sorry I missed the review period on this, but I don't think CSS for a reconciliation dialog should be modifying *any* of the common styles. In my opinion, none of the modifications to `.dialog-footer`, `.button-left`, and `.button-right` don't belong here. The problem is that this file is always loaded, so the definitions here are going to overwrite the intended definitions where they are first defined.

Perhaps @ayushrai206 can clarify the intent for the original changes.

Comment (wetneb) [reply main/webapp/modules/core/styles/reconciliation/recon-dialog.css:105]: Sorry that it escaped me in the review, it's worth fixing indeed!

**Pull request merged**